from exceptions._base_exception import BaseException


class BaseEntityException(BaseException):
    pass
